client/verta/verta/endpoint/build/_build.py

```diff
@@ -97,6 +99,35 @@ def message(self) -> str:
     def is_complete(self) -> bool:
         return self.status in ("finished", "error")
 
+    @property
+    def nvidia_gpu_compatible_hardware(self):
```

Comment:
Ughh...maybe let's just remove this? We've tucked away "hardware compatibility" as a concept deep inside `endpoint.update()` so we're not really obligated to expose this (like you mentioned before).

And Eric is likely to implement an `endpoint.get_resources()` as a part of pipelines, so we can expose "here is the GPU you ordered" there.